darwin/DarwinProcess.c

```diff
@@ -263,8 +263,8 @@ void DarwinProcess_setFromLibprocPidinfo(DarwinProcess *proc, DarwinProcessList
 
       proc->super.time = (pti.pti_total_system + pti.pti_total_user) / 10000000;
       proc->super.nlwp = pti.pti_threadnum;
-      proc->super.m_size = pti.pti_virtual_size / 1024 / PAGE_SIZE_KB;
-      proc->super.m_resident = pti.pti_resident_size / 1024 / PAGE_SIZE_KB;
+      proc->super.m_size = pti.pti_virtual_size / 1024 / CRT_pageSizeKB;
+      proc->super.m_resident = pti.pti_resident_size / 1024 / CRT_pageSizeKB;
```

Comment:
`/ CRT_pageSize` instead of `/ 1024 / CRT_pageSizeKB`

Reply:
Well spotted, changed